src/accelerate/utils/environment.py

```diff
@@ -13,7 +13,21 @@
 # limitations under the License.
 
 import os
-from distutils.util import strtobool
+
+
+def str_to_bool(value) -> bool:
+    """
+    Converts a string representation of truth to `True` (1) or `False` (0).
+
+    True values are `y`, `yes`, `t`, `true`, `on`, and `1`; False value are `n`, `no`, `f`, `false`, `off`, and `0`;
+    """
+    truth = ["y", "yes", "t", "true", "on", "1"]
+    false = ["n", "no", "f", "false", "off", "0"]
+
+    if value.lower() not in truth + false:
+        raise ValueError(f"{value} is not a valid boolean value")
+
+    return value.lower() in truth
```

Comment:
Is there a specific reason to change the signature from int to bool? `str_to_bool(value) == 1` still works since bool is a subclass of int, but it can be confusing.

Reply:
Reverted